highfive/newpr.py

```diff
@@ -195,15 +202,13 @@ def choose_reviewer(repo, owner, diff, exclude):
         return 'test_user_selection_ignore_this'
 
     # Get JSON data on reviewers.
-    with open(repo + '.json') as rf:
-        reviewers = json.load(rf)
+    reviewers = json.loads(_load_json_file(repo + '.json'))
```

Comment:
yup. FWIW, the function was originally meant to load more than just json. I'm planning to refactor most of this code soon. I'll move it, though.